src/gluonts/ev/ev_tutorial.md

```diff
@@ -0,0 +1,162 @@
+# New evaluation approach
+
+This notebook explains how to use the `ev` module which is a new alternative to the `evaluation` module in GluonTS.
+
+## [Usage](usage_example.py)
+
+Before diving into how things work, let's look at how to use the new evaluation approach.
+
+### Overview
+1. Decide on the `test_data` (of type `TestData`) and `predictor` to use.
+2. Gather the `metrics` to be evaluated and decide over what `axis` to aggregate (use `None` to aggregate to a single value).
+3. Call `predictor.backtest(metrics, test_data, axis)` to get (metric name, metric result) pairs.
+
+### Example
+
+```python
+from gluonts.dataset.split import split
+from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.model.npts import NPTSPredictor
+from gluonts.ev.metrics import MSIS, MSE, SumQuantileLoss
+```
+
+First, let's pick a dataset. We choose `exchange_rate` here as it's very lightweight.
+
+```python
+dataset = get_dataset("exchange_rate")
+
+prediction_length = dataset.metadata.prediction_length
+freq = dataset.metadata.freq
+```
+
+To get the (input, label) pairs of type `TestData`, we do the following:
+
+```python
+_, test_template = split(dataset=dataset.test, offset=-prediction_length)
+test_data = test_template.generate_instances(prediction_length=prediction_length)
+```
+
+Let's use a non-parametric predictor to keep things simple for this example:
+```python
+predictor = NPTSPredictor(prediction_length=prediction_length, freq=freq)
+```
+
+Next, we specify what metrics to evaluate. We don't provide arguments to MSE and MSIS because we are fine with the default values (in this case, taking the mean forecast as reference for MSE and using `alpha=0.05` for MSIS).
+
+```python
+metrics_per_entry = [MSE(), SumQuantileLoss(q=0.9), MSIS()]
+```
+
+To evaluate these metrics, we call the `backtest` method on our predictor. We want to have these metrics evaluated per time series so we specify `axis=1 `(an `axis` of `0` would aggregate per timestamp and `None` would aggregate overall). The `num_samples` is an optional argument used during the call to `predict`.
+
+```python
+evaluation_result = predictor.backtest(
+    metrics=metrics_per_entry,
+    test_data=test_data,
+    axis=1,
+    num_samples=100,
+)
+```
+
+Finally, we print the `evaluation_result`.
+
+```python
+for name, value in evaluation_result.items():
+    print(f"\n{name}: {value}")
+```
+
+This gives:
+```python
+MSE: [2.42091992e-03 2.96873403e-04 2.32838181e-04 4.56364661e-04
```

Comment:
I'm not sure how this is helpful, beyond just showing some "scientific" numbers.